src/utils/action-handler.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+import { Router } from "express";
+import Promise from "bluebird";
+
+import responseMiddleware from "./response-middleware";
+import requireHullMiddleware from "./require-hull-middleware";
+
+export default function actionHandler(handler) {
+  const router = Router();
+  router.use(requireHullMiddleware);
+  router.post("/", (req, res, next) => {
+    return Promise.resolve(handler(req.hull, _.pick(req, ["body", "query"]))).then(next, next);
```

Comment:
I found it quite confusing if I have `fn(req.hull, req)` as arguments, since I can get first from the second one. What would you tell about:

`handler(req.hull, _.omit(req, ["hull"]))`?